source/iterators.tex

```diff
@@ -6520,6 +6523,28 @@
 \pnum \returns \tcode{N}.
 \end{itemdescr}
 
+\indexlibrary{\idxcode{ssize(C\& c)}}%
+\begin{itemdecl}
+template <class C>
+constexpr auto ssize(const C& c)
+    -> common_type_t<ptrdiff_t, make_signed_t<decltype(c.size())>>;
+\end{itemdecl}
+\begin{itemdescr}
+\pnum \returns
+\begin{codeblock}
+static_cast<common_type_t<ptrdiff_t, make_signed_t<decltype(c.size())>>>(c.size())
```

Comment:
Append a period (I realize this is in a codeblock, but it's really a sentence).

Reply:
Sorry, that wouldn't be adequate, because the period would be in code font inside a codeblock. I don't think we do that elsewhere.

Reply:
Yes, you're right.  I assuming you're using codeblocks to prevent an Overfull?  What we tend to do in this case is split the expr over two lines with a \tcode{} for the part of the expr that should go on each line.  If you want the whole expression on one line, would it work to tcode the whole expr, prefixing it with a newline and an indent?  In any case, what follows \returns is supposed to be a sentence, while codeblocks should be used for complete code snipets.

Reply:
According to the [Specification Style Guidelines](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#writing-returns-in-a-function-description), codeblocks are also used for long expressions.

Reply:
@johelegp Thanks for pointing that out!  I don't remember seeing that the last time I read that page.  Is there a way to "see what's new" there?

Reply:
If you're watching the repository, you should be able to see when a new there's a new [revision](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines/_history). It should appear in your GitHub dashboard. For more visibility, I follow my GitHub feed through RSS.